airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -272,8 +272,9 @@ def _start_template_dataflow(self, name, variables, parameters,
         # Builds RuntimeEnvironment from variables dictionary
         # https://cloud.google.com/dataflow/docs/reference/rest/v1b3/RuntimeEnvironment
         environment = {}
-        for key in ['maxWorkers', 'zone', 'serviceAccountEmail', 'tempLocation',
-                    'bypassTempDirValidation', 'machineType', 'network', 'subnetwork']:
+        for key in ['numWorkers', 'maxWorkers', 'zone', 'serviceAccountEmail',
```

Comment:
Maybe you could also add the "additionalExperiments" array. Seems to be the last missing out of the keys  defined in https://cloud.google.com/dataflow/docs/reference/rest/v1b3/RuntimeEnvironment . Anything preventing us from doing so ?

Reply:
@potiuk You're right. I missed that one.